modules/countdown.js

```diff
@@ -14,5 +14,17 @@ module.exports = {
         return 'Python 2.7 support ends in ' + moment('2020-01-01T00:00:00Z').countdown().toString()
       }
     }
+  },
+  onload: {
+    function (bot) {
+      function autoCount (bot, lastTick) {
+        var thisTick = moment('2019-03-29T23:00:00Z').countdown().toString().split(/, | and /)[0]
+        if (thisTick !== lastTick) {
+          bot.say('Article 50 expires in ' + thisTick)
+        }
+        setTimeout(autoCount(bot, thisTick), 1000)
```

Comment:
This will run `autoCount()` before running `setTimeout() `, and as `autoCount` returns undefined, this will end up setting the timeout for nothing to a thousand milliseconds.